modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -5,16 +5,11 @@
   "States": {
     "ExtractFilepaths": {
       "Parameters": {
-        "cma": {
-          "event.$": "$",
-          "ReplaceConfig": {
-            "FullMessage": true
-          },
-          "task_config": {
-            "buckets": "{$.meta.buckets}",
-            "fileBucketMaps": "{$.meta.collection.files}",
-            "excludedFileExtensions": "{$.meta.collection.meta.orca.excludedFileExtensions}"
-          }
+        "event.$": "$",
```

Comment:
Not sure if this will work as you expect. The parametes contains input, etc. see https://docs.aws.amazon.com/step-functions/latest/dg/input-output-inputpath-params.html we may want to think about what we call thesse in the future and depending on what you find, we may want to discuss during ORCATecture the format we would like.

Reply:
Pending discussion during ORCAtechture on what we should use as parameter keys.

Reply:
updated to 
`"input.$": "$.payload"`